src/components/Carousel.js

```diff
@@ -372,8 +393,15 @@ class Carousel extends Component {
             handledDelta = 0;
         }
 
-        const position = currentPosition + (100 / (this.state.itemSize / handledDelta)) + '%';
-
+        let position = currentPosition + (100 / (this.state.itemSize / handledDelta));
```

Comment:
I'm still trying to get my head around a few things. Would it be OK to ask you to add some comments around some areas to help? Here I would estimate this:
```
// When allowing infinite loop, if we slide left from position 0 we reveal the cloned last slide that appears before it - if we slide even further we need to jump to other side so it can continue - and vice versa for the last slide
```